engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -99,4 +100,12 @@ package object engine {
       case _ => actorSystem.stop(materializeWorkflowDescriptorActor)
     }
   }
+
+  final case class EngineWorkflowDescriptor(// TODO: Re-add:: declarations: WorkflowCoercedInputs
```

Comment:
Would it make sense to add this todo as a ticket w/ details so we don't forget what it was?